config/vufind/searches.ini

```diff
@@ -542,6 +542,10 @@ sort = "last_indexed desc"
 ;       call numbers using the custom CallNumber search handler.
 ; Tag
 ;       Provide suggestions from the local database of tags.
+; SolrPrefix:[Autocomplete field]:[Display field]
+;       Perform a search against autocomplete field with edge-n gram tokenizer,
```

Comment:
I have some thoughts on making this comment a little more explicit, but before I suggest revised text, let's finish discussing the other details of the PR to make sure I understand the intent correctly.

Reply:
How about something like:

Perform a search against [Autocomplete field] using the edge n-gram tokenizer. Each word in the user's query is handled as a prefix. Values from [Display field] will be displayed for matching records. It is recommended that you index values for this handler into fields of type "text_autocomplete" (which are available using the dynamic field suffix *_autocomplete). These values should also be copied into a stored field (or a field with docValues) for display purposes.

Reply:
One question, though: is the last sentence actually accurate? Should it be "string field" rather than "stored field"?

Reply:
Yes, field must be stored or with docValues so faceting on that field is working.

Reply:
Of course, makes sense. So perhaps the comment could be revised to:

Perform a search against [Autocomplete field] using the edge n-gram tokenizer. Each word in the user's query is handled as a prefix. Values from [Display field] will be displayed for matching records. It is recommended that you index values for this handler into fields of type "text_autocomplete" (which are available using the dynamic field suffix *_autocomplete). These values should also be copied into a stored field (or a field with docValues) for display purposes, because display of values by this option relies on faceting.

Reply:
Actually, wait a moment, maybe it doesn't entirely make sense -- I thought faceting ignored stored values and always used indexed values instead. Am I mistaken?

Reply:
No, you are right, field used for faceting is not required to be stored. Should I fix the comments accordingly?

Reply:
Sure. Maybe the easiest solution is to change the last sentence of my proposed revision to something like "These values should also be copied into a separate field for display purposes, because display of values by this option relies on faceting -- be sure to choose an appropriate field type, such as a simple _str field, to ensure correct display."

Reply:
OK, I fixed the comments in the latest commit.

Reply:
Thanks, looks good!